phpstorm-url-handler

```diff
@@ -8,12 +8,12 @@
 # @author Stefan Auditor <stefan.auditor@erdfisch.de>
 
 arg=${1}
-pattern=".*file(:\/\/|\=)(.*)&line=(.*)"
+pattern=".*file(?::\/\/|\=)(.*)&line=(.*)"
```

Comment:
I tested the merged code and unfortunately it failed.
Non-capturing groups are not supported by `sed`
That regex syntax is only supported in PCRE (not BRE or ERE which sed supports)